src/query/custom_cursors/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(mg_custom_cursors_sources
+  once.cpp
+  scanall.cpp
+  produce.cpp
+)
+add_library(mg_custom_cursors STATIC ${mg_custom_cursors_sources})
+# TODO(gitbuda): Having mg-storage-v2 here is broken.
```

Comment:
Broken because
<img width="1358" alt="Screenshot 2024-03-09 at 10 18 41 AM" src="https://github.com/gitbuda/memgraph/assets/4950251/acdeefd5-e361-426a-b477-499c18b2dfb7">